admin/src/main/resources/io/buoyant/admin/js/src/router_clients.js

```diff
@@ -40,6 +40,8 @@ define([
         return initializeClient(client);
       });
 
+      routerClients = _.compact(routerClients);
```

Comment:
tioli: `_.chain()` this to the map in the previous line so these don't get separated